app/scripts/modules/google/src/certificate/certificate.reader.ts

```diff
@@ -20,9 +20,10 @@ export class GceCertificateReader {
 
   public listCertificates(): IPromise<IGceCertificate[]> {
     return this.searchService
-      .search<IGceCertificate>({ q: '', type: 'sslCertificates' }, this.infrastructureCaches.get('certificates'))
+      .search<IGceCertificate>({ q: '***', type: 'sslCertificates' }, this.infrastructureCaches.get('certificates'))
       .then((searchResults: ISearchResults<IGceCertificate>) => {
         if (searchResults && searchResults.results) {
+          debugger;
```

Comment:
I don't know what this is?

Reply:
Pauses execution and opens a debugger in chrome if you have the console open -- need to remove :)